pkg/repo/repo.go

```diff
@@ -30,26 +35,41 @@ func GetProfileDefinition(repoURL, branch, path string) (profilesv1.ProfileDefin
 		return profilesv1.ProfileDefinition{}, errors.New("unsupported git provider, only github.com is currently supported")
 	}
 
-	profileURL := strings.Replace(repoURL, "github.com", "raw.githubusercontent.com", 1)
-	profileURL = fmt.Sprintf("%s/%s/%s/profile.yaml", profileURL, branch, path)
-
-	resp, err := httpClient.Get(profileURL)
```

Comment:
the checking to see if its `github.com` a few lines above can probably go too :smile:

Reply:
Yeah. :D